### PR TITLE
[TLC] admin accountabilities of HR cirlce

### PR DESCRIPTION
**Pull request by cassus**
As requested in slack from Petak

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,6 +205,10 @@ Purpose:
 - Satisfied partners
 - A-class people
 
+Accountabilities:
+- Scanning IDs of new hires to https://drive.google.com/drive/u/0/folders/0B_dmDpfClsCFWlNfRE55Tnh2SlU
```
Comment (peterlangmar) [inline README.md]: new comer should do this as the profess is faster

File: README.md [hunk 2]
```diff
@@ -205,6 +205,10 @@ Purpose:
...
+- Adding personal data of new hires to https://docs.google.com/a/brickflow.com/document/d/1pUMIv8rYsxvQ6-oiGw7ieaxIMLxSIXZREr_edReSR3c/edit?usp=sharing
```
Comment (peterlangmar) [inline README.md]: like

File: README.md (hunk 1 above)
Comment (cassus) [inline README.md]: The circle can still decide who will do.

This is a marked accountability as the Admin role will hold the HR role accountable if the IDs are not uploaded.

Comment (peterlangmar) [inline README.md]: As I think the new comer should do it, the HR role should tell this to him / her. Should be part of the general onboarding checklist.

**Pull request merged**